README.md

```diff
@@ -1,24 +1,25 @@
 # id.webmaker.org
+> OAuth 2.0 identity provider for Webmaker
 
 [![Build Status](https://travis-ci.org/mozilla/id.webmaker.org.svg?branch=master)](https://travis-ci.org/mozilla/id.webmaker.org)
+[![Code Climate](https://codeclimate.com/github/mozilla/id.webmaker.org/badges/gpa.svg)](https://codeclimate.com/github/mozilla/id.webmaker.org)
+[![David-DM](https://david-dm.org/mozilla/id.webmaker.org.svg)](https://david-dm.org/mozilla/id.webmaker.org)
 
-OAuth 2.0 identity provider for Webmaker
+`id.webmaker.org` is an application with a [`hapi`]( http://hapijs.com) backend and [`react`](http://facebook.github.io/react) frontend that serves as the OAuth 2.0 identity provider for [Webmaker](http://webmaker.org), as well as several other Mozilla Foundation applications.
 
-## To run
+## Prerequisites
 
-```
-npm install
-npm start
-```
-
-Make sure you configure all necessary environment variables (see below) before running. If you simply want the defaults set up for local development, you can use:
+- [Node](https://nodejs.org) and NPM, [installation](https://github.com/nodejs/node-v0.x-archive/wiki/Installing-Node.js-via-package-manager)
+- [Postgres](http://www.postgresql.org/), [installation](http://www.postgresql.org/download/)
 
-```
-$> cp sample.env .env
-```
-
-Once, after cloning the repo but before running the code.
+## Up and Running
```

Comment:
i like `up and running` and have used it for most oss projects i have. any particular reason you don't like it?